der/Cargo.toml

```diff
@@ -19,9 +19,9 @@ rust-version = "1.65"
 arbitrary = { version = "1.3", features = ["derive"], optional = true }
 bytes = { version = "1", optional = true, default-features = false }
 const-oid = { version = "0.9.2", optional = true }
```

Comment:
Could that be an opportunity to bump this dependency as well?
(I believe you published const-oid yesterday?)

Reply:
Saving it for a followup due to #1293.

I'm going to make some API changes to it first before upgrading it.

Reply:
Also upgrading needs prereleases of a bunch of the hash and cipher crates, due to their use of OIDs.